7.2/data_input.txt

```diff
@@ -0,0 +1,11 @@
+12/31/2015:john.a.graham@gmail.com
+10/31/2001:a.marie.gay@aol.com
+5/12/2014:coit.scharringhausent@juno.com
+8/24/1996:fflinstone@aol.com
+9/9/1985:kat4ever01@hotmail.com
+10/10/2002:regisfilia@fake.aol.com
+01/25/1984:coit125@aol.com
+04/20/1983:jobethegreat@hotmail.com
```

Comment:
add a line where the email is just plain missing, and another line where the date is just plain missing.